pyproject.toml

```diff
@@ -30,7 +30,7 @@ python = "^3.11"
 requests = "^2.31.0"
 llama-cpp-python = "^0.2.56"
 drain3 = "^0.9.11"
-progressbar2 = ">=3.53.2"
+huggingface-hub = "^0.23.2"
```

Comment:
This is not packaged for Fedora :(

Reply:
whoops :/ we forgot about this

the package itself looks okay, but the extras are really demanding

https://github.com/huggingface/huggingface_hub/blob/main/setup.py

Reply:
Do we need extras?

Reply:
No we don't. HF treats this as "almost" a metapackage, different types for different use cases. For our purposes we can stick with minimal subset of functionality and things will work. I'll confirm this and get back to you though, just to be sure.